sdk/search/search-documents/test/public/node/searchIndexClient.spec.ts

```diff
@@ -18,36 +18,34 @@ import {
 import { delay, serviceVersions } from "../../../src/serviceUtils";
```

Comment:
I think this test file will also be getting replaced by an empty file, since there is an internal reference here. @HarshaNalluru

Reply:
Not doing it here to keep the changes minimal